website/content/partials/cli/monitor/flags/log-format.mdx

```diff
@@ -0,0 +1,9 @@
+<a id="monitor-flag-log-format" />
+
+**`-log-format (string: "standard")`**
```

Comment:
```suggestion
**`-log-format (string : "standard")`**
```

The new layout adds a space between the type and the colon to improve accessibility and scanning